src/common/common.h

```diff
@@ -97,13 +97,17 @@
  *  and is responsible for freeing it. */
 #define OWNER
 
+/** Structs that are passed over the wire should be annotated with this
+ *  attribute to make sure there is no padding. */
+#define SERIALIZABLE __attribute__((packed))
```

Comment:
you are just talking about the macro name, right? I think it's good to keep the name a bit more general, because we may want to add more attributes, like aligned(8), for instance. So it's not just about tight packing.